qemu/tests/cfg/chardev_free_port.cfg

```diff
@@ -0,0 +1,9 @@
+- chardev_free_port:
+    only Linux
+    type = chardev_free_port
+    serials = " vs2 "
+    serial_type_vs2 = virtserialport
+    chardev_backend_vs2 = socket
+    extra_params = ' -qmp tcp:0:4444,server,nowait ' 
```

Comment:
No need.

Reply:
I think the test need this parameter or else it cannot get correct chardev info

Reply:
@michiboo  Not exactly,  you do not need to add extra qmp, framework have default one, it can be used to get the chardev info.